Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -46,6 +46,8 @@ void Processor::initialize()
     dmesgln("CPU[{}]: Supports {}", m_cpu, build_cpu_feature_names(m_features));
     dmesgln("CPU[{}]: Physical address bit width: {}", m_cpu, m_physical_address_bit_width);
     dmesgln("CPU[{}]: Virtual address bit width: {}", m_cpu, m_virtual_address_bit_width);
+    if (!has_feature(CPUFeature::RNG))
+        dmesgln("CPU[{}]: {} not detected, randomness will be poor", m_cpu, cpu_feature_to_description(CPUFeature::RNG));
```

Comment:
Not sure if this dmesgln is necessary, since we already print `KernelRng: No entropy source available!` if no random source is used in the random number generator.

Reply:
taken that directly from x86-64; this is slightly different as is related only to CPU, it seems that all raspberry pi’s have hardware rng — using that you’d have only this message but not the one from `KernelRng`…

of course i more than open for message changes (also their content)!

Reply:
Ah okay, I wasn't aware that x86_64 was also printing this. It's fine then :)